src/lib/groovy/modsdotgroovy/DependenciesBuilder.groovy

```diff
@@ -35,15 +37,35 @@ import static groovy.lang.Closure.DELEGATE_FIRST
 @CompileStatic
 class DependenciesBuilder extends HashMap {
 
-    private List<Dependency> dependencies = []
+    private List<Dependency> forgeDependencies = []
+    private List<Dependency> quiltDependencies = []
+
+    void onQuilt(@DelegatesTo(value = DependenciesBuilder, strategy = DELEGATE_FIRST)
+               @ClosureParams(value = SimpleType, options = "modsdotgroovy.DependenciesBuilder") final Closure closure) {
+        final builder = new DependenciesBuilder()
+        closure.delegate = builder
+        closure.resolveStrategy = DELEGATE_FIRST
+        closure.call(builder)
+        quiltDependencies.addAll(builder.build().quilt)
+    }
+
+    void onForge(@DelegatesTo(value = DependenciesBuilder, strategy = DELEGATE_FIRST)
```

Comment:
Please add similar `setForge` functionality for QSL / QLoader.

Reply:
QSL isn't a single mod, so I'm not quite sure how to do it for that, but QLoader should be easy

Reply:
Well, the quilt loader stuff is implemented. I'm still struggling a bit with what the best approach to QSL and quilted-fabric-api is here